modules/nodes-nodes-viewing-listing.adoc

```diff
@@ -106,13 +106,17 @@ $ oc describe node node1.example.com
 ----
 Name:               node1.example.com <1>
 Roles:              worker <2>
-Labels:             beta.kubernetes.io/arch=amd64   <3>
-                    beta.kubernetes.io/instance-type=m4.large
+Labels:             beta.kubernetes.io/os=linux
```

Comment:
Not sure if this is valid, we will need to change this to `kubernetes.io/os`

https://v1-26.docs.kubernetes.io/docs/reference/labels-annotations-taints/#beta-kubernetes-io-os-deprecated -> source link